modules/help_editor/test/help_editorTest.php

```diff
@@ -153,6 +156,9 @@ function testHelpEditorSearchByTopic()
      */
     function testHelpEditorSearchByKeyword()
     {
+         $this->markTestSkipped(
+             'Modifications needed to adapt to layout changes'
+         );
```

Comment:
@kongtiaowang This test was giving me trouble and I'm not sure how to resolve it.

Reply:
@HenriRabalais OK, keep this mark here, I will fix this after this PR get merged.